src/lib/features/project/project-store.ts

```diff
@@ -539,20 +539,28 @@ class ProjectStore implements IProjectStore {
     async getProjectsByUser(userId: number): Promise<string[]> {
         const projects = await this.db
             .from((db) => {
-                db.select('project')
+                db.select('role_user.project')
                     .from('role_user')
                     .leftJoin('roles', 'role_user.role_id', 'roles.id')
+                    .leftJoin('projects', 'role_user.project', 'projects.id')
                     .where('user_id', userId)
+                    .andWhere('projects.archived_at', null)
```

Comment:
other archived_at usages are behind a flag but since we're just about to remove the flag from the code I add it directly here